src/app/home-page/widget/entry-box/entry-box.component.ts

```diff
@@ -82,19 +80,22 @@ export class EntryBoxComponent extends Base implements OnInit {
   }
 
   getMyEntries() {
-    this.userEntries$ = this.usersService.getUserEntries(null, this.filterText, this.entryTypeParam);
-    this.userEntries$
+    this.usersService
+      .getUserEntries(null, this.filterText, this.entryTypeParam, 'response')
       .pipe(
         finalize(() => (this.isLoading = false)),
         debounceTime(750),
         takeUntil(this.ngUnsubscribe)
       )
       .subscribe(
-        (myEntries: Array<EntryUpdateTime>) => {
-          myEntries.forEach(() => {
-            this.listOfEntries = myEntries.slice(0, 7);
-            this.totalEntries = myEntries.length;
-          });
+        (myEntries: HttpResponse<EntryUpdateTime[]>) => {
+          this.listOfEntries = myEntries.body.slice(0, 7);
+          // Update total entries only when no search filter applied (i.e. non-filtered total)
```

Comment:
Why are we only updating the total entries when there's no search filter? Wouldn't it be helpful if we displayed the number of search results?

Reply:
This is true and really up to preference, https://github.com/dockstore/dockstore-ui2/pull/1713#discussion_r1110367559 is why we originally decided to keep `totalOrgs`/ `listOfEntries` static.

But in turn, the UI displays `listOfEntries`, so now that it's static, it doesn't change... whether we want the UI to display the the number of search results instead is up for debate @coverbeck @denis-yuen 

I can change `this.listOfEntries = myEntries.body.slice(0, 7);` to `this.listOfEntries = myEntries.body;` then display `this.listOfEntries.length` in the bubble as the new total and use the slice pipe in the `*ngFor` to display the first 7 entries.

Just checked the mocks [here](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/6022abd3355a9d26154a8d95) and the search bar is handled a bit differently, with the results being in a dropdown and the total displayed being unchanged. Whether we want to go with that option is also up for discussion.

![image](https://user-images.githubusercontent.com/97123241/222827904-f9ffef21-e401-402e-b8a5-db91ff4e66ac.png)

Reply:
> This is true and really up to preference, https://github.com/dockstore/dockstore-ui2/pull/1713#discussion_r1110367559 is why we originally decided to keep totalOrgs/ listOfEntries static.

My comment was pointing out that you could end up displaying a confusing "You are in no organizations" when the user does belong to at least one org. I took no stance there on which way to solve it. :) 

But I do lean towards displaying the total count.

Since we're discussing it, the other confusing thing that the mocks don't cover is what portion of the filtered results you are seeing, i.e., if the search finds 10 results, we only display 7 and don't tell you there are more. In this case, I like what we do in the search page, with the "x more", although I don't think in this case we need to make it an expandable link. Plus it's not in the mocks.

![Screen Shot 2023-03-03 at 1 21 47 PM](https://user-images.githubusercontent.com/1049340/222831833-11c0ffe0-db8f-42be-a165-c4808f93267c.png)

In summary:

* I lean towards displaying the total
* I'd like it if we showed how many the filter is matching; if it's more than 7, show how many matches aren't being shown, otherwise just show the matches.

Which I guess is saying I'd like to see both :).

But I don't feel super strongly, and could be easily convinced either way.

Reply:
> Just checked the mocks [here](https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/6022abd3355a9d26154a8d95) and the search bar is handled a bit differently, with the results being in a dropdown and the total displayed being unchanged. Whether we want to go with that option is also up for discussion.

I like it :slightly_smiling_face: feels like the best of both worlds, you can see the total number of entries, and also all of the search results in the drop down. I think currently, you don't see all search results

Reply:
Those are all really great points, I guess I also agree, the mocks are probably the best of both worlds

Reply:
Just playing around, going with the dropdown idea, any preference on introducing a scroll bar to limit the number of search results shown or display the total results list no matter the length?

We can also style the scroll to be less of an eye sore

![image](https://user-images.githubusercontent.com/97123241/222840384-450ef0c4-85fa-4f1a-a90a-92f93ee41f28.png)

Reply:
I think the mock-scroll-bar is more of an auto-complete than a way of showing results (i.e. would probably skip the dates). Otherwise, appreciate the conversation so far and don't have too much to add.